### PR TITLE
Gimbal: Added rc channel control readme

### PR DESCRIPTION
**Pull request by snktshrma**
In reference to the [PR](https://github.com/ArduPilot/ardupilot_gazebo/pull/102) by @srmainwaring. Updated readme to setup and control gimbal over ground control.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +232,44 @@ Display the streamed video:
 gst-launch-1.0 -v udpsrc port=5600 caps='application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264' ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink sync=false
 ```
 
+View the streamed camera frames in OGC:
+
+`Open QGC > Application Settings > Video Settings > Select UDP h.264 Video Stream & use port 5600`
```
Comment (srmainwaring) [inline README.md:237]: Add a screen shot of the QGC settings panel?

File: README.md [hunk 2]
```diff
@@ -232,6 +232,44 @@ Display the streamed video:
 gst-launch-1.0 -v udpsrc port=5600 caps='application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264' ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink sync=false
 ```
 
+View the streamed camera frames in OGC:
```
Comment (srmainwaring) [inline README.md:235]: typo: OCG -> QCG. Make a hyperlink to the relevant QCG help page.

File: config/gazebo-iris-gimbal.parm [hunk 3]
```diff
@@ -14,17 +14,21 @@ RNGFND1_SCALING  10
...
+RC7_OPTION      213
+RC8_OPTION      214
 
-# Gimbal roll and pitch servos
-SERVO5_FUNCTION  8
-SERVO6_FUNCTION  7
+# Gimbal roll, pitch and yaw servos
+SERVO9_FUNCTION  8
+SERVO10_FUNCTION  7
+SERVO11_FUNCTION  6
```
Comment (srmainwaring) [inline config/gazebo-iris-gimbal.parm:34]: Add blank line at end.

File: README.md [hunk 4]
```diff
@@ -232,6 +232,44 @@ Display the streamed video:
...
+
+`Open QGC > Application Settings > Video Settings > Select UDP h.264 Video Stream & use port 5600`
+
+### 4. Using Gimbal
```
Comment (srmainwaring) [inline README.md:239]: maybe: Using the 3d gimbal

File: README.md [hunk 5]
```diff
@@ -232,6 +232,44 @@ Display the streamed video:
...
+
+To control gimbal using RC channels,
```
Comment (srmainwaring) [inline README.md:241]: Suggestion: The Iris model is equipped with a 3d gimbal and camera that can be controlled directly in MAVProxy using the RC overrides.

**Review by srmainwaring — CHANGES_REQUESTED** [review]

**Comment by srmainwaring** [comment]
Thanks for the PR @snktshrma. Looks good. Have suggested a few changes. A screenshots of the QGC settings page might help clarify the setup, but not essential.

**Comment by snktshrma** [comment]
Hi @srmainwaring! I have resolved the requested changes and added a screenshot of QGC settings page!

**Comment by rmackay9** [comment]
@snktshrma, Thanks for this.  The QGC image isn't appear for me and the url looks a bit suspicious (https://private-user-images.githubusercontent.com/74557164/344478908-f55242f3-cff8-4f81-befd-da68e4df73c6.png).

**Comment by snktshrma** [comment]
Hi @rmackay9 sir! I have updated the image! Hopefully it's visible now

**Comment by srmainwaring** [comment]
Adding resized image here so it can be referenced from as an asset in the main repo.

qgc_video_settings

**Review by srmainwaring — APPROVED** [review]
Looks good thanks @snktshrma. I've rebased to bring the parameters up to date with `main` and referenced the QGC image from this repo rather than your fork.

**Pull request merged**